README.md

```diff
@@ -70,7 +71,30 @@ Before beginning a sprint:
+
+When planning a sprint, the team determines how many total points they should commit to by multiplying their **average-sprint-velocity** by the *total person-hours* available in the sprint they are planning. The result is the total number of points the team should commit to. If every member of the team will be present for the entire sprint, the total points are divided evenly to each member; every individual’s contribution is considered equal. If a team member will only be able to contribute to half of a sprint, their **individual-velocity** will be 50% of the total number of hours in the sprint multiplied by the **average-sprint-velocity**. The individual velocity factors in the specific team member’s available hours and always relies on the average velocity for the team; it is never reliant on the individual’s previous velocities and individual velocity is never recorded at the end of a sprint.
+
+> **individual-velocity** = *hours the team member can contribute to the sprint* * **average-sprint-velocity**
+
+Velocity tracking has several benefits - it increases predictability, leads to more detailed tasks, and ensures all members of the team have an amount of work they feel comfortable with. Since **average-sprint-velocity** is the average of previous sprints, commitment should end with no member having too much to complete or too little to complete as it is based on how much work the team was able to finish in the past. The average also increases predictability by greatly improving the likelihood that the team will finish what they said they would finish. This prevents blockers because all dependencies of a task will have been resolved in a previous sprint.
+
+Velocity tracking improves task definition because well defined tasks are a prerequisite to estimation. When estimating a task, the point value assigned is based on time, complexity, and unknowns. Reducing the number of unknowns makes the point value more accurate because it removes ambiguity and allows the team to point solely on the task’s time and complexity. The point values the team chooses from should get progressively larger because as a task gets bigger, the number of unknowns increases. A good scale to use for point values is the fibonacci sequence (1, 2, 3, 5, 8, 13). In the example of the fibonacci sequence, task estimates should remain below a 13. If a task gets larger than an 8, that means that it is too vague and should broken down into several more specific tasks with their own point values. Team members should estimate each task as a group and converge on a point value that every member agrees with. This ensures that each task is pointed consistently and flushes out any additional unknowns. It doesn’t matter if two separate teams use a different scale, as long as they are consistent within their own team.
```

Comment:
Should we dedicate an earlier section to task estimation and only highlight the benefits of velocity tracking on task estimation here?